EIPS/eip-1193.md

```diff
@@ -54,27 +52,6 @@ Multiple RPC protocols may be available. For examples, see:
 - [EIP-1474](https://eips.ethereum.org/EIPS/eip-1474), the Ethereum JSON-RPC API
 - [EIP-1767](https://eips.ethereum.org/EIPS/eip-1767), the Ethereum GraphQL schema
 
-### sendAsync (DEPRECATED)
```

Comment:
I really liked the idea of documenting the history of send/sendAsync since nobody else has and it is extremely confusing.

Reply:
Moved to appendix.